guides/source/plugins.md

```diff
@@ -455,6 +455,22 @@ gem "yaffle", git: "https://github.com/rails/yaffle.git"
 After running `bundle install`, your gem functionality will be available to the application.
 
 When the gem is ready to be shared as a formal release, it can be published to [RubyGems](https://rubygems.org).
+
+Alternatively, you can benefit from Bundler's Rake tasks. You can see a full list with the following:
+
+```
+rake -T
+
+rake build
+# Build yaffle-0.0.1.gem into the pkg directory
```

Comment:
The version is probably `0.1.0`.  Ref: https://github.com/rails/rails/commit/c3a9e97f848e27a4e6875879a24388d9af0d608f